assets/js/util/index.js

```diff
@@ -907,6 +907,8 @@ export const showErrorNotification = ( ErrorComponent, props = {} ) => {
 export const decodeHtmlEntity = ( str ) => {
 	const decoded =  str.replace( /&#(\d+);/g, function( match, dec ) {
 		return String.fromCharCode( dec );
+	} ).replace( /(\\)/g, function( ) {
+		return '';
```

Comment:
No need for a function here, this can simply become `.replace( /(\\)/g, '' )`

Reply:
Much better, thanks @felixarntz!